src/components/scan/hooks/useFileUpload.ts

```
@@ -0,0 +1,150 @@
+}
+
+interface UseFileUploadOptions {
+	documentConfig?: DocumentConfig;
+	onUploadSuccess?: () => void;
+	onUploadStart?: () => void;
+	onUploadComplete?: () => void;
+}
+
+interface UseFileUploadReturn {
+	isUploading: boolean;
+	uploadDocumentFile: (file: File, importedFrom: string) => Promise<any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)